test/controllers/admin/assignees_controller_test.rb

```diff
@@ -0,0 +1,43 @@
+require 'test_helper'
+
+class Admin::AssigneesControllerTest < ActionController::TestCase
+  attr_reader :user, :application, :lawyer
+
+  setup do
+    @user = users(:admin)
+    @application = applications(:in_progress)
+    @lawyer = users(:lawyer)
+
+    sign_in_as(@user)
+  end
+
+  test "POST #create associates lawyer to application and makes primary" do
+    assert_difference 'application.assignees.count' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.